### DISCUSSION TITLE
Deployments suddenly stopped working

### DISCUSSION THREAD

**Discussion by lt7** [vercel/community#5392] [Help] [answered]
### Summary

My project of a few months has just stopped creating new deployment today. I pushed a new feature branch and no deployment was triggered, PR that commit to a dev branch and no deployment triggered either.

Attempted to deploy manually the feature branch commit gives a 'Not Found' error and the PR to dev branch has just stuck at this for past ~10 mins or so, builds usually complete in under 3 mins. 

<img width="451" alt="Screenshot 2024-01-08 212824" src="https://github.com/vercel/community/assets/1315232/b6c05d82-a5e5-4672-aef9-8d349189da14">

### Steps to Reproduce

Unknown

**Comment by Wojciech-Szczepanski** [comment]
We've encountered a similar problem in our project. We've been stuck in the build step for approximately 2 hours now. We cannot cancel or redeploy this build, trigger another build from GitHub Actions, or contact support because this option is blocked.
![Screenshot 2024-01-08 at 12 23 10](https://github.com/vercel/community/assets/112784039/99753fdd-8139-4c83-967b-a322785fc869)
![Screenshot 2024-01-08 at 12 23 30](https://github.com/vercel/community/assets/112784039/5bc0cbe4-c634-42a3-8cf3-408497c5300f)

**Comment by ishaan1995** [comment]
Getting a similar issue today, no new deployment created for any new branch / main branch
Tried from vercel cli to deploy but got an error: 
```
Error: An unexpected internal error occurred
```
<img width="478" alt="image" src="https://github.com/vercel/community/assets/12032343/fa765a93-2ba1-46d0-9ba2-190fc82ee092">

**Comment by dominikmlynarczyk** [comment]
Hello Vercel team (@leerob, @swarnava, @amyegan),

The same **critical** problem also occurred in our project.
We are unable to trigger any build, due to the fact that the problematic build has crashed for over 4 hours:
![blurred-vercel](https://github.com/vercel/community/assets/40466173/36e53e9a-b9aa-485a-842f-eef7db60c929)

Can't remove deployment via CLI or vercel platform:
<img width="453" alt="image" src="https://github.com/vercel/community/assets/40466173/e4dfe5b7-8019-4d1f-b933-c90adedc7a6b">
**Due to this bug, we currently have a completely blocked CI/CD process in the project.**

**Comment by swarnava** [comment]
Hi there,

Sorry about the inconvenience. Our team is currently investigating this and share update here: https://www.vercel-status.com/incidents/p1zsvb8vgnjz

**Comment by swarnava** [comment, marked as answer]
Hi,

A fix has been implemented and we are monitoring the results.
